src/xcode/ENA/ENAUITests/AntigenTestProfile/ENAUITests_12_AntigenTestProfile.swift

```diff
@@ -150,9 +150,15 @@ class ENAUITests_12_AntigenTestProfile: CWATestCase {
 		snapshot("antigentestprofile_screenshot_edit_actions")
 
 		/// Edit Antigen Test Profile Screen
-
-		let editTestProfileAction = try XCTUnwrap(app.sheets.buttons[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Profile.editAction])
-		editTestProfileAction.waitAndTap(.extraLong)
+		
+		let editTitle: String
+		if Locale.current.identifier == Locale(identifier: "de_DE").identifier {
+			editTitle = "Schnelltest-Profil bearbeiten"
```

Comment:
Why not use: `AccessibilityLabels.localized(AppStrings.AntigenProfile.Profile.editActionTitle)` to get the correct identifier?

Reply:
Yeah this just worked thanks!, I pushed again, hope it works on the CI as well